Framework/Geometry/inc/MantidGeometry/Objects/MeshObject.h

```diff
@@ -202,7 +202,7 @@ class MANTID_GEOMETRY_DLL MeshObject : public IObject {
   boost::shared_ptr<vtkGeometryCacheWriter> m_vtk_cache_writer;
 
   /// Contents
-  std::vector<int> m_triangles;
+  std::vector<size_t> m_triangles;
```

Comment:
Since you are providing these as constructor arguments. You can make these const now, particularly if copy-assignment is disabled too. i.e. `const std::vector<size_t> m_triangles;

Reply:
Could you also add some better comments around this `m_triangles` and `m_vertices` are linked. `m_triangles` indexes into `m_vertices`.